### ISSUE TITLE
esp-idf can't link liblvgl.a (IDFGH-5154)

### ISSUE THREAD

**Issue by mfp20** [espressif/esp-idf#6932]
## Environment

- Development Kit:      M5Stack Fire
- Kit version (for WroverKit/PicoKit/DevKitC): NA
- Module or chip used:  ESP32 (unspecified)
- IDF version (run ``git describe --tags`` to find it): v4.3-beta2-2-g9a2d25191
- Build System: idf.py (CMake)
- Compiler version (run ``xtensa-esp32-elf-gcc --version`` to find it): 8.4.0
- Operating System:     Linux
- Using an IDE?: No (vim)
- Power Supply: USB+Battery

## Problem Description

Detailed description with logs at LVGL: https://github.com/lvgl/lvgl/issues/2213

In short: the linker command line includes liblvgl.a and the static lib has the required syms but the linker throw 'undefined reference' errors.

**Comment by igrr** [comment]
The issue has been moved to https://github.com/lvgl/lv_port_esp32/issues/274. If the problem still persists, please consider replying there.
